APE0.rst

```diff
@@ -0,0 +1,198 @@
+The Astropy Project Bylaws
+==========================
+
+author: The Astropy Governance Working Group: Tom Aldcroft, Matt Craig, Kelle Cruz, Lia Corrales, Steve Crawford, Nicole Foster, Pey Lian Lim, Stuart Mumford, Adrian Price-Whelan, Thomas Robitaille, David Shupe, Brigitta Sipocz, Erik Tollerud
+
+date-created: 2020 July 31
+
+date-last-revised: 2020 July 31
+
+date-accepted: <replace with accepted date>
+
+type: Process
+
+status: Discussion
+
+Abstract
+========
+This APE is a living document that defines the formal fundamental governance process for the Astropy Project. Governance is based around the election of a Coordination Committee. The Coordination Committee has broad authority, which they seek to exercise as rarely as possible by working towards consensus.
+
+
+Detailed description
+====================
+
+The bylaws are described in this section. The initial version is the outcome of several months of discussion by the Astropy Governance Working Group, which aimed to both codification of existing informal governance rules as well as update it with new ideas that are consistent with the philosophy of and vision for the Astropy project.
+
+
+Ultimate Authority: The Astropy Community
+-----------------------------------------
+At its core, the Astropy Project derives its authority from the whole community - developers, users, and even potential participants in the broader astronomical and scientific software community. All have a vested interest in the success of the Project, so all governance in the end belongs to this community in its entirety.  The governance bylaws described below provide levels of delegation of that authority, particularly to the voting members and the Coordination Committee.  The Astropy Project broadly follows the philosophy of do-ocracy: project participants who gain experience and knowledge through sustained community contributions and have demonstrated adherence to core principles of inclusion and consensus-building will be entrusted with increasing levels of authority. At its core, the Project is both by and for its community, so it is the final source of authority and direction.
+
+The Coordination Committee
+--------------------------
+
+Composition
+^^^^^^^^^^^
+The Coordination Committee is a committee that is composed of four members. To represent a broad set of interests, the Coordination Committee should have no more than two members from a particular employer. The Ombudsperson (described below) and Coordination Committee should work to maintain this composition following the processes laid out in these bylaws.
+
+Mandate
+^^^^^^^
+The Coordination Committee’s role is to facilitate consensus in the Project and coordinate Project-spanning efforts.  As such, the Coordination Committee shall work to:
+* Maintain the quality and stability of the Astropy ecosystem of packages for the community,
```

Comment:
All these itemized lists need a blank line  in front to be rendered properly.

Reply:
You're right!  The danger of trusting in "automatic" conversion-tools... thanks, will fix